connect_four/connect_four.rb

```diff
@@ -1 +1,73 @@
-# Your code here!
\ No newline at end of file
+require 'pry'
+require 'pry-byebug'
+require './board.rb'
+require './player.rb'
+require './computer.rb'
+
+
+class Game
+
+  def initialize
+    @player1 = Player.new("X")
+    @player2 = Player.new("O")
+    @current_player = @player1
+    @game_board = Board.new
+  end
+
+  # The main play loop
+  def play
+    welcome
```

Comment:
`play` could be incorporated into `initialize` so the game starts with a single comment, e.g. `game = Game.new` and you're off and running.